plugins/providers/gcs/provider.go

```diff
@@ -202,38 +195,12 @@ func validateProviderConfigAndAppealParams(pc *domain.ProviderConfig, a *domain.
 	return nil
 }
 
-func getPermissions(resourceConfigs []*domain.ResourceConfig, a *domain.Appeal) ([]Permission, error) {
-	var resourceConfig *domain.ResourceConfig
-	for _, rc := range resourceConfigs {
-		if rc.Type == a.Resource.Type {
-			resourceConfig = rc
-		}
-	}
-	if resourceConfig == nil {
-		return nil, ErrInvalidResourceType
-	}
-
-	var role *domain.Role
-	for _, r := range resourceConfig.Roles {
-		if r.ID == a.Role {
-			role = r
-		}
-	}
-	if role == nil {
-		return nil, ErrInvalidRole
-	}
-
+func getPermissions(a *domain.Appeal) []Permission {
```

Comment:
is this function specific for gcs or a generic one?

could this go as function receiver in `domain.Appeal`? so calling this would be `a.GetPermission()`
cc @rahmatrhd

Reply:
it's a generic function(there in other providers as well), not sure about the second part of the question. @rahmatrhd

Reply:
This helper function is provider specific, I don't think it requires to be considered as a receiver function in domain.Appeal. @rahmatrhd

Reply:
this is specific to gcs because it cast permission string into `gcs.Permission` type